src/main/java/frc/robot/subsystems/hang/docs/Hang.md

```diff
@@ -0,0 +1,51 @@
+# Hang subsystem
+## description
+This subsystem allows the frc robot to hang on the chain in the stage.
+
+
+##connected devices
+a motor (can sparkmax)
+
+
+## Methods
+| Method Name | Returns | Parameters | Description | Notes |
+| --- | --- | ---- | ---- | --- |
+|getInstance|instance of hang |N/A|creates a new version of the hang|    |
+|hang|void|N/A|once instance is made, creates a constructor for hang|   |
+|setupMotors|void|N/A|starts initial code.| |
```

Comment:
write a bit more (eg. sets the current limit, inversions, conversion factors of the hang motor)